frontend/packages/ceph-storage-plugin/src/components/independent-dashboard-page/details-card/card.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { referenceForModel, K8sResourceKind } from '@console/internal/module/k8s';
+import { SubscriptionModel } from '@console/operator-lifecycle-manager';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+import { FirehoseResource, FirehoseResult } from '@console/internal/components/utils';
+import { getName, getInfrastructurePlatform } from '@console/shared';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import DetailItem from '@console/shared/src/components/dashboard/details-card/DetailItem';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import { useK8sGet } from '@console/internal/components/utils/k8s-get-hook';
+import { InfrastructureModel } from '@console/internal/models';
+import { getOCSVersion } from '../../../selectors';
+import { OCSServiceModel } from '../../../models';
+
+const k8sResources: FirehoseResource[] = [
+  {
+    kind: referenceForModel(OCSServiceModel),
+    namespaced: true,
+    isList: false,
+    namespace: 'openshift-storage',
+    prop: 'ocs',
+  },
+  {
+    kind: referenceForModel(SubscriptionModel),
+    namespaced: false,
+    isList: true,
+    prop: 'subscription',
+  },
+];
+
+const DetailsCard = React.memo(
+  withDashboardResources((props: DashboardItemProps) => {
+    const { watchK8sResource, stopWatchK8sResource, resources } = props;
+
+    React.useEffect(() => {
+      k8sResources.forEach((r) => watchK8sResource(r));
+      return () => {
+        k8sResources.forEach((r) => stopWatchK8sResource(r));
+      };
+    }, [watchK8sResource, stopWatchK8sResource]);
+
+    const { ocs, subscription } = resources;
+    const ocsLoaded = _.get(ocs, 'loaded', false);
```

Comment:
optional chaining is preferred - same for all other instances of `_.get`
```suggestion
    const ocsLoaded = ocs?.loaded
```